Error while checking build/O2Physics/staging for 5f0fe3b42d42d9e26083091d0d0ea69b434cac68 at 2026-08-20 01:02:
```
## sw/BUILD/O2Physics-latest/log
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGLF/Utils/CascadeMlResponse.h:58:7: error: enumeration value 'eta' not handled in switch [-Werror=switch]
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGLF/Utils/CascadeMlResponse.h:58:7: error: enumeration value 'eta' not handled in switch [-Werror=switch]
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/O2Physics/17520/5f0fe3b42d42d9e26083091d0d0ea69b434cac68/build_O2Physics_staging/pretty.html).

--- a/PWGLF/TableProducer/Strangeness/cascademlselection.cxx
+++ b/PWGLF/TableProducer/Strangeness/cascademlselection.cxx
@@ -16,6 +16,7 @@
 /// \author David Dobrigkeit Chinellato <david.dobrigkeit.chinellato@cern.ch>, Austrian Academy of Sciences
 //
 
+#include "Tools/ML/MlResponse.h"
 #include "PWGLF/DataModel/LFStrangenessMLTables.h"
 #include "PWGLF/DataModel/LFStrangenessTables.h"
 #include "PWGLF/Utils/CascadeMlResponse.h"
@@ -42,6 +43,10 @@
 #include <memory>
 #include <string>
 #include <vector>
+#include <iterator>
+
+#include "TH1.h"
+#include "TString.h"
 
 using namespace o2;
 using namespace o2::framework;

--- a/PWGLF/TableProducer/Strangeness/cascademlselection.cxx
+++ b/PWGLF/TableProducer/Strangeness/cascademlselection.cxx
@@ -16,10 +16,11 @@
 /// \author David Dobrigkeit Chinellato <david.dobrigkeit.chinellato@cern.ch>, Austrian Academy of Sciences
 //
 
-#include "Tools/ML/MlResponse.h"
 #include "PWGLF/DataModel/LFStrangenessMLTables.h"
 #include "PWGLF/DataModel/LFStrangenessTables.h"
 #include "PWGLF/Utils/CascadeMlResponse.h"
+
+#include "Tools/ML/MlResponse.h"
 
 #include <CCDB/BasicCCDBManager.h>
 #include <CCDB/CcdbApi.h>
@@ -35,18 +36,18 @@
 #include <Framework/OutputObjHeader.h>
 #include <Framework/runDataProcessing.h>
 
+#include <TH1.h>
+#include <TString.h>
+
 #include <algorithm>
 #include <cmath>
 #include <cstdint>
 #include <cstdlib>
+#include <iterator>
 #include <map>
 #include <memory>
 #include <string>
 #include <vector>
-#include <iterator>
-
-#include "TH1.h"
-#include "TString.h"
 
 using namespace o2;
 using namespace o2::framework;

--- a/PWGLF/Utils/CascadeMlResponse.h
+++ b/PWGLF/Utils/CascadeMlResponse.h
@@ -37,8 +37,7 @@ enum class InputFeaturesCasc : uint8_t {
   dcaV0daughters,
   dcav0topv,
   bachBaryonCosPA,
-  bachBaryonDCAxyToPV,
-  eta
+  bachBaryonDCAxyToPV
 };
 
 template <typename TypeOutputScore = float>

